### PR TITLE
Update punctuation in some docs

### PR DESCRIPTION
**Pull request by whangho**
Updating some grammar and punctuation in docs/README.md and docs/reST/ref/freetype.rst.

### CODE REVIEW

File: docs/reST/ref/freetype.rst [hunk 1]
```diff
@@ -54,7 +54,7 @@ The :any:`pygame.examples.freetype_misc <pygame.examples.freetype_misc.main>`
 example shows these features in use.
 
 The pygame package does not import ``freetype`` automatically when
-loaded. This module must be imported explicitly to be used. ::
```
Comment (MyreMylar) [inline docs/reST/ref/freetype.rst:57]: I think those double colons are part of the .rst syntax for introducing a code block IIRC.  Unless that has changed at some point.

Comment (MyreMylar) [reply docs/reST/ref/freetype.rst:57]: Yeah I just tested - removing these breaks the code block.

File: docs/reST/ref/freetype.rst [hunk 2]
```diff
@@ -54,7 +54,7 @@ The :any:`pygame.examples.freetype_misc <pygame.examples.freetype_misc.main>`
...
+loaded. This module must be imported explicitly to be used.
```
Comment (MyreMylar) [inline docs/reST/ref/freetype.rst:57]:
```suggestion
loaded. This module must be imported explicitly to be used. ::
```

File: docs/reST/ref/freetype.rst [hunk 3]
```diff
@@ -459,7 +459,7 @@ loaded. This module must be imported explicitly to be used. ::
       passed to a previous :meth:`render_to`, :meth:`get_rect`, :meth:`render`,
       :meth:`render_raw`, or :meth:`render_raw_to` call. Primarily, this
       feature is an aid to using :meth:`render_to` in combination with
-      :meth:`get_rect`. An example: ::
+      :meth:`get_rect`. An example:
```
Comment (MyreMylar) [inline docs/reST/ref/freetype.rst:462]:
```suggestion
      :meth:`get_rect`. An example: ::
```

File: docs/reST/ref/freetype.rst (hunk 1 above)
Comment (whangho) [reply docs/reST/ref/freetype.rst]: Ah, I didn't realize! Sorry, clearly I'm still learning. Appreciate your review here!

**Review by damusss — APPROVED** [review]
LGTM :)

**Pull request merged**